FreeScape.Engine/Providers/TextureProvider.cs

```diff
@@ -45,6 +45,14 @@ private Texture CreateAndAddTexture(string textureName, string gtl, TextureInfo
             return texture;
         }
 
+        public void AddTexture(Texture texture, string gtl)
```

Comment:
This is a hack for now so I can focus on the animation design rather than dealing with textures for hours